draft/spec/index.html

```diff
@@ -340,6 +340,47 @@ <h2>Object Structure</h2>
         ├── inventory.json
         └── inventory.json.sha512
         </pre>
+
+        <section id="object-interoperability">
+            <h2>Interoperability</h2>
+            <p>
+                Within an OCFL Object and its associated <a>Inventory</a>, the following restrictions are applied
+                to filesystem features. These restrictions are provided in an attempt to normalize file handling
+                behaviors and addressability of files within the Inventory.
+            </p>
+            <p>
+                If use of these features are required to preserve an exact copy of the content (for example,
+                preserving a hard-drive image), implementers SHOULD consider using a packaging format,
+                such as ZIP or TAR files or ISO disk images, to preserve the content. See the
+                [[OCFL-Implementation-Notes]] for further details.
+            </p>
+            <ul>
+                <li>
+                    <strong>Symbolic links</strong> (aka 'shortcuts' or 'aliases') MUST NOT be used within an
+                    OCFL Object. These are not portable and may lead to undefined behaviours in client applications
+                </li>
+                <li>References to <strong>file paths</strong> within the <a>inventory</a> MUST use the "forward-slash"
+                    ('/') as a path separator. Some filesystem use other characters such as the "back-slash" ('\')
+                    or "colon" (':') as a path delimiter.
+                </li>
+                <li>
+                    <strong>File permissions</strong> MAY be applied to files in an OCFL Object; however, implementers
```

Comment:
File permissions are unavoidably applied to any file in all current filesystems I know about so the MAY here seems odd. I also don't think we should introduce fuzzy terms like ACLs and hidden files. I think we should make a simpler statement along the lines of:

> File permissions are not portable across filesystems and are not expected to be preserved by OCFL clients.

Reply:
@rosy1280 I think the edge case to consider is when a repository is rebuilt on a file system that handles filenames differently than the filesystem the OCFL Object was created on. In that situation (as in the example with BagIt) it might be possible for the filenames in the inventory to differ from the actual filenames (even though they both look the same, visually).